panel/tests/ui/template/test_fastlisttemplate.py

```diff
@@ -25,7 +25,9 @@ def test_fast_list_template_no_console_errors(page):
         "[bokeh] setting log level to: 'info'",
         "[bokeh] Websocket connection 0 is now open",
         "[bokeh] document idle at",
-        "Bokeh items were rendered successfully"
+        "Bokeh items were rendered successfully",
+        "[bokeh] JS/Python version mismatch",
```

Comment:
Maybe the last additions here should be behind a specific version check of bokeh 3.4.0. Could in the future be a real error.

Reply:
```suggestion
        "[bokeh] JS/Python version mismatch (3.4.0-rc.",
```